pkg/api/v1/types.go

```diff
@@ -1732,3 +1732,12 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items" description:"list of component status objects"`
 }
+
+// ServiceIPAllocation is not a public type
+type ServiceIPAllocation struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Excellent! verify-description.sh failed. Thanks for verifying that it is no longer broken. :-)